Dependencies/AndroidExtensions/Source/JavaWrappers.cpp

```diff
@@ -388,3 +407,23 @@ namespace android::net
         return {env->CallStaticObjectMethod(cls, env->GetStaticMethodID(cls, "parse", "(Ljava/lang/String;)Landroid/net/Uri;"), (jstring)uriString)};
     }
 }
+
+namespace android::graphics
+{
+    SurfaceTexture::SurfaceTexture()
+        : Object("android/graphics/SurfaceTexture")
+    {
+    }
+
+    void SurfaceTexture::initWithTexture(int texture)
+    {
+        m_object = m_env->NewObject(m_class, m_env->GetMethodID(m_class, "<init>", "(I)V"), texture);
+    }
+
+    void SurfaceTexture::updateTexture() const
+    {
+        if (m_object) {
```

Comment:
If you make this set by the constructor (as described in previous comments), then you don't need this check, which will then also more closely match what the rest of the JavaWrappers code does.